connector/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/v2/PostgresIntegrationSuite.scala

```diff
@@ -123,4 +134,77 @@ class PostgresIntegrationSuite extends DockerJDBCIntegrationV2Suite with V2JDBCT
       )
     }
   }
+
+  override def testDatetime(tbl: String): Unit = {
```

Comment:
just for my curiosity, what does `super.testDatatime` do? an empty function?

Reply:
Yes. Just is an empty function.